cert_manager/__init__.py

```diff
@@ -128,7 +129,20 @@ def request_missing(self, configured_domains, current_domains):
                 timestamp = datetime.utcnow().timestamp()
                 dns_updated = domain_group['dns_records_updated']
                 if dns_updated is not None and timestamp - dns_updated > self.dns_delay:
-                    self.request_cert(domains)
+                    internal_domains = []
+                    external_domains = []
+                    for each_domain in domains:
+                        domain_elem = get_tld(each_domain, as_object=True)
+                        # Separate internal domains and external domains
+                        if domain_elem.domain == 'opencraft':
```

Comment:
Nice one @pkulkark!

We can go one step further and make it completely generic - loop through all domains and group them by top domain. Then we wouldn't have to hardcode 'opencraft' and in theory be able to support multiple domains.